docs/src/main/sphinx/admin/properties-general.md

```diff
@@ -67,3 +67,16 @@ size limits.
 Prepared statement compression is not applied if the size gain is less than the
 configured value. Smaller statements do not benefit from compression, and are
 left uncompressed.
+
+## `spatial-geometry-acceleration-degree`
+
+- **Type:** {ref}`prop-type-string`
+- **Allowed values:** `MILD`, `MEDIUM`, `HOT`
+- **Default value:** `MILD`
+- **Session property:**  `spatial_geometry_acceleration_degree`
+
+Set the degree of acceleration of the geometry. Acceleration usually builds a
+raster and a quadtree. `MILD` acceleration, takes least amount of memory.
+`MEDIUM` acceleration, takes more memory and takes more time to accelerate,
+but may work faster. `HOT` acceleration, takes even more memory and may take
+longest time to accelerate, but may work faster than the other two.
```

Comment:
Does it mean that HOT is more suited for long queries while MILD for short queries?

i.e. it's unclear what "time to accelerate" means.

Reply:
Here `time to accelerate` means `time required for the optimization process`.

> `HOT` acceleration could result in the highest memory consumption and potentially the longest time required for the optimization process. However, once the acceleration is complete, operations involving geometries are expected to work very quickly.

Reply:
Is there any mention of terms "geometry acceleration" elsewhere in the docs? This page has nothing on the topic and might be confusing without at least links to related docs.